styles/media-queries.css

```diff
@@ -2,26 +2,93 @@
 /*  */
 /*}*/
 
-/*@media (max-width: 1280px) {*/
+@media (max-width: 1280px) {
+
+}
+
+/*@media (max-width: 1024px) {*/
 
 /*}*/
 
-@media (max-width: 1024px) {
-  .footer__list {
-    width: 150px;
+@media (max-width: 768px) {
+  .logo__header {
+    width: 122px;
+    height: 24px;
   }
-}
 
-@media (max-width: 768px) {
-  .footer__list {
-    justify-content: center;
+  .advantages {
+    padding-top: 60px;
+    padding-bottom: 61px;
+  }
+
+  .footer {
+    padding-top: 60px;
+    padding-bottom: 60px;
   }
+
+
 }
 
 @media (max-width: 480px) {
+  .advantages {
```

Comment:
I suppose it's better to use a generic class like container or wrapper. As you can see, all block gaps are equal. So making one generic class would be the best idea.